core/tests/protractor_desktop/contributorDashboard.js

```diff
@@ -148,7 +148,7 @@ describe('Contributor dashboard page', function() {
     await contributorDashboardPage.navigateToSubmitQuestionTab();
     await contributorDashboardPage.waitForOpportunitiesToLoad();
     await contributorDashboardPage.expectOpportunityWithPropertiesToExist(
-      SKILL_DESCRIPTIONS[0], TOPIC_NAMES[0], null, '(0.00%)');
+      SKILL_DESCRIPTIONS[0], TOPIC_NAMES[0], null, '(0%)');
```

Comment:
Why this change? Ditto elsewhere.

Reply:
This change is needed because now in the front-end we show percentages without decimals.